src/types/props.ts

```diff
@@ -245,6 +245,13 @@ export interface PropsBase {
    * @see https://daypicker.dev/docs/customization#showweeknumber
    */
   showWeekNumber?: boolean;
+  /**
+   * Experimental. Animate navigating between months.
+   *
+   * @experimental
```

Comment:
Interesting take. Not sure if this should be classified as "experimental".

```suggestion
   * Animate navigating between months.
```

Reply:
All good for me, this part I just copied and pasted from your other PR, so I left the way it was